buildenv/jenkins/testJobTemplate

```diff
@@ -264,7 +264,21 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							    sectionHeader('Choose which platform to run tests on, and optionally specify machine label to send the test job to a certain machine is desired.')
 							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
 							}
-							stringParam('PLATFORM', "${ARCH_OS}", "Platform to be tested")
+							stringParam('PLATFORM', "${ARCH_OS}", '''Platform to be tested:
+                                <br>aarch64_linux
+                                <br>ppc64_aix
+                                <br>ppc64le_linux
+                                <br>s390x_linux
+                                <br>s390x_zos
+                                <br>x86-64_linux
+                                <br>x86-64_mac
+                                <br>x86-64_windows
+                                <br>x86-32_windows
+                                <br>sparcv9_solaris
+                                <br>riscv64_linux
+                                <br>multiple platforms (comma separated values): x86-64_linux,x86-64_mac
+                                <br>all platforms: all (actual platforms will change based on JDK_VERSION/JDK_IMPL)
+                                <br>Please refer to PLATFORM_MAP in https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/openjdk_tests<br>''')
```

Comment:
```suggestion
                                <br/>aarch64_linux
                                <br/>ppc64_aix
                                <br/>ppc64le_linux
                                <br/>s390x_linux
                                <br/>s390x_zos
                                <br/>x86-64_linux
                                <br/>x86-64_mac
                                <br/>x86-64_windows
                                <br/>x86-32_windows
                                <br/>sparcv9_solaris
                                <br/>riscv64_linux
                                <br/>multiple platforms (comma separated values): x86-64_linux,x86-64_mac
                                <br/>all platforms: all (actual platforms will change based on JDK_VERSION/JDK_IMPL)
                                <br/>Please refer to PLATFORM_MAP in https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/openjdk_tests<br/>''')
```

Reply:
I am also wondering about the quotes around this string, I think they can remain as double quotes " rather than the 3 single quotes, but I am happy to run a test to verify it

Reply:
I saw similar usage of these quotes for multi-line description in [L312-L323](https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/testJobTemplate#L312-L323) so I decided to follow that too. I am okay with testing it to verify and see if this works :)

Reply:
yes, so if you push an update to use `<br/>` in place of `<br>` tags, I will run a test (keeping the ''').